tests/manage/mcg/test_bucket_creation_deletion.py

```diff
@@ -148,14 +148,37 @@ class TestBucketCreation(MCGTest):
             "1-CLI-PVPOOL",
         ],
     )
-    def test_bucket_creation(
-        self, bucket_class_factory, bucket_factory, amount, interface, bucketclass_dict
+    def test_bucket_creation_deletion(
```

Comment:
After combining the creation and deletion tests, how much improvement did we see in the run time?

Reply:
Mentioned in the description now